test/unit/MCR/updateMCR.js

```diff
@@ -323,23 +166,4 @@ describe('updateMCR', function () {
     const expectedDesiredMCR = totalSumAssured.mul(10000).div(gearingFactor);
     expect(desiredMCR).to.be.equal(expectedDesiredMCR);
   });
-
-  it('decreases desiredMCR towards mcrFloor when poolValueInEth = 0 and totalSumAssured = 0', async function () {
```

Comment:
what is the desiredMCR under these circumstances now, that we don't have a floor? do we end up having issues like division by 0? do they have any impact?

Reply:
The new desiredMCR becomes directly the gearedMCR